.github/scripts/get-tags-to-push.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+function find_last_matching_version() {
+  FILTER=$1
+  git tag | grep -v BETA | grep '^v' | cut -c2- | grep "^$FILTER" | tail -n 1
+}
+
+function get_latest_version() {
+  find_last_matching_version ""
+}
+
+function verlte() {
+    [  "$1" = "`echo -e "$1\n$2" | sort -V | head -n1`" ]
+}
+
+VERSION_TO_RELEASE=$1
+MINOR_VERSION_TO_RELEASE=${VERSION_TO_RELEASE%.*}
+MAJOR_VERSION_TO_RELEASE=${MINOR_VERSION_TO_RELEASE%.*}
+
+LATEST_FOR_MINOR=$(find_last_matching_version $MINOR_VERSION_TO_RELEASE)
+LATEST_FOR_MAJOR=$(find_last_matching_version $MAJOR_VERSION_TO_RELEASE)
+LATEST=$(get_latest_version)
+
+TAGS_TO_PUSH+=($VERSION_TO_RELEASE)
+
+if verlte "$LATEST_FOR_MINOR" "$VERSION_TO_RELEASE"; then
+   TAGS_TO_PUSH+=($MINOR_VERSION_TO_RELEASE)
+fi
+
+if verlte "$LATEST_FOR_MAJOR" "$VERSION_TO_RELEASE"; then
+   TAGS_TO_PUSH+=($MAJOR_VERSION_TO_RELEASE)
+fi
+
+if verlte "$LATEST" "$VERSION_TO_RELEASE"; then
+   TAGS_TO_PUSH+=(latest)
+fi
+
+echo "${TAGS_TO_PUSH[@]}"
```

Comment:
Could we add a test for this file? I tried couple of examples and it mostly works ok, but e.g. for 

```
$ ./get-tags-to-push.sh 5.3.0-BETA-3
5.3.0-BETA-3 5.3 5 latest
```

I think tags `5` and `latest` shouldn't be pushed for beta, and maybe 5.3 also, but not sure there.

Reply:
> Could we add a test for this file?

Added https://github.com/hazelcast/hazelcast-docker/pull/554/commits/938969d0570ec6c4e11f0feeb84bd1ca62afb0d9

> I think tags 5 and latest shouldn't be pushed for beta

I agree, fixed: https://github.com/hazelcast/hazelcast-docker/pull/554/commits/f48f084672b497aeaa514df0c58910d778fc4fc1